runtime/near-vm-runner/src/logic/logic.rs

```diff
@@ -2043,7 +2051,7 @@ impl<'a> VMLogic<'a> {
     /// * `yield_create_byte` for each byte of `method_name` and `arguments`;
     /// * Fees for reading the `method_name` and `arguments`;
     /// * Fees for writing the Data ID to the output register;
-    /// * Fees for setting up the receipt.
+    /// * Fees for setting up the receipt and the continuation action therein.
```

Comment:
I don't mind this language "continuation action" but I don't think we use it elsewhere. If we aim to lean on the user's understanding of standard promise function calls to help them understand the yielded promises, we should use consistent language between them.